papaparse.js

```diff
@@ -643,6 +644,15 @@ License: MIT
 			}
 
 			xhr.open(this._config.downloadRequestBody ? 'POST' : 'GET', this._input, !IS_WORKER);
+			if (this._config.downloadTimeout) {
+				timeout = parseInt(this._config.downloadTimeout);
+				if (!isNaN(timeout)) {
+					xhr.timeout = timeout;
+					xhr.ontimeout = bindFunction(function() { this._chunkError('timeout'); }, this);
+				} else {
+					this._chunkError('Config downloadTimeout value (' + this._config.downloadTimeout + ') not parsable by parseInt(val).');
```

Comment:
I'm not sure if it really makes sense to raise an error or just ignore the non Numeric timeout.

Reply:
I disagree. Why shouldn't the user be informed of misuse of the API?

Reply:
Maybe but using a wrong timeout will not cause any parsing error.

Reply:
You're right - but the user would unknowingly get incorrect behavior from the library (because `timeout` is not set). Being silently in an incorrect state is worse than failing completely, in my opinion.

Reply:
Then probably better to raise and exception but not a parsing one.

Reply:
Ah, right - absolutely! I agree with you on that.

Reply:
@pokoli I moved the check into Parser (where it  belongs) and I saw that in other cases no error is raised. So I believe that to keep things uniform no Error should be raised in case of invalid input. My personal opinion is that a user must always be informed of invalid values in configuration. Anyway it's your call.

Reply:
@demetris-manikas If you prefer to inform of invalid configuration it should not be a parse execption and we should warn for all invalid operations. But this is out of the scope of this PR (i prefer to keep them minumum)

Feel free to create a separate PR that adds it for all the current arguments.